packages/dataclass-editor/src/master/useValidation.ts

```diff
@@ -0,0 +1,7 @@
+import { useAppContext } from '../context/AppContext';
+import type { DataClassField } from '../data/dataclass';
+
+export const useValidation = (field: DataClassField) => {
```

Comment:
I would directly add the name as param here and not the field. But just a personal opinion. This is fine too :)